### ISSUE TITLE
[BUG] Reeder app stops working after update to 1.18

### ISSUE THREAD

**Issue by kriskenyon** [FreshRSS/FreshRSS#3531]
After updating to 1.18 Reeder prompted for login again.  Using the API login at end point api/greader.php/accounts/ClientLogin
I am getting the following return
SID=user/largehex
Auth=same
It then calls api/greader.php/reader/api/0/user-info?output=json
which returns Unauthorized!

**Environment information (please complete the following information):**
Apache 2.4.43
Mysql 8
PHP 7.4.6 

This worked right until the update then failed. 
I confirmed Reeder is sending the Authorization header with the correct Auth return.

**Comment by Zeik0s** [comment]
Hm, interesting, for me Reeder works, but i'm using it out of the docker container on a raspi 4. 
And since i didn't export the data as a volume when i created the old 1.17 i had to recreate the container, so didn't really update from 1.17 to 1.18.

**Comment by Alkarex** [comment]
Could you please try via command line? (replacing with your own account information)
https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html#google-reader-compatible-api

```sh
curl 'https://freshrss.example.net/api/greader.php/accounts/ClientLogin?Email=alice&Passwd=Abcdef123456'

curl -v -H "Authorization:GoogleLogin auth=alice/8e6845e089457af25303abc6f53356eb60bdb5f8" \
  'https://freshrss.example.net/api/greader.php/reader/api/0/user-info?output=json'
```

Have you tried to disconnect and reconnect again in the app?

**Comment by kriskenyon** [comment]
I tried curl here is the result
HTTP/1.1 401 Unauthorized
< Date: Mon, 15 Mar 2021 23:40:19 GMT
< Server: Apache
< X-Powered-By: PHP/7.4.6
< Google-Bad-Token: true
< Transfer-Encoding: chunked
< Content-Type: text/plain; charset=UTF-8
redacted host
Unauthorized!* Closing connection 0

I have tried setting up this as fresh and I never get past the authentication from api/greader.php/reader/api/0/user-info?output=json

FYI I was watching reeder in a proxy tool to inspect the traffic. and it was getting the same results.

**Comment by Alkarex** [comment]
You have a strange `Google-Bad-Token: true` in your response. Maybe an interfering proxy or something.

* Try the built-in test on https://freshrss.example.net/api/
* Try to set again your API password in FreshRSS
* Try again the two cURL commands (does the first cURL command work?)

**Comment by kriskenyon** [comment]
I have reset my api password a few times both to the same and different. 
The 1st curl command works as does the initial login in Reeder.
The second command fails in both.
I created a new user and that user can login so it seems there is something in my users config.php at error but I cam not sure what that should look like?

**Comment by kriskenyon** [comment]
Ok I got this working after narrowing it down to the account. I have the following errors in my user log
is_admin does not exist in configuration
enabled does not exist in configuration
It appears that somewhere in the upgrade process these settings did not get applied. I have one other user and confirmed there account also is missing this. As soon as I added that back in to /data/users/name/config.php it started working again.
